core/src/main/java/uk/ac/bbsrc/tgac/miso/core/util/UnicodeBOMInputStream.java

```diff
@@ -68,7 +68,7 @@
  * Use the {@link #skipBOM()} method to remove the detected BOM from the wrapped <code>InputStream</code> object.
  * </p>
  */
-public class UnicodeBOMInputStream extends InputStream {
```

Comment:
It probably doesn't matter, but should probably remain public just because "why not?"